src/com/paynemiller/ch3/SimpleInterestComputer.java

```diff
@@ -0,0 +1,38 @@
+package com.paynemiller.ch3;
+
+import java.util.Scanner;
+
+public class SimpleInterestComputer {
+    Scanner keyboard = new Scanner(System.in);
+    double principalAmount;
+    double rateOfInterest;
+    int numYears;
+    double simpleInterest;
+
+    public void promptForInput(){
+        System.out.print("Enter the principal: ");
+        principalAmount = keyboard.nextDouble();
+        System.out.print("Enter the rate of interest: ");
+        rateOfInterest = keyboard.nextDouble();
+        System.out.print("Enter the number of years: ");
+        numYears = keyboard.nextInt();
+    }
+
+    public void calculateSimpleInterest(){
+        rateOfInterest = rateOfInterest / 100;
+        simpleInterest = principalAmount * (1 + (rateOfInterest * numYears));
+        simpleInterest = Math.round(simpleInterest * 100.0) / 100.0; // rounds to the next decimal
+    }
+
+    public void showSimpleInterest(){
+        System.out.print("After " + numYears + " years at " + rateOfInterest * 100 + "%, the investment will be worth $" + simpleInterest);
+    }
+
+    public void output(){
+        promptForInput();
+        calculateSimpleInterest();
+        showSimpleInterest();
+    }
+
+    //formatting could be fixed
```

Comment:
IntelliJ [can auto-format](https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html) it for you!

Reply:
Fixed the formatting issues.

https://github.com/paynemiller92/programming-practice-erick/pull/12/commits/d45185acb0be4e3b0c4c364bdb8f3edc78e79b20